packages/mystmd/tests/dois/outputs/dois.bib

```diff
@@ -60,7 +60,7 @@ @article{Koch_2012
 
 @article{EFSA2019Dietary,
 	address = {IT},
-	author = {{EFSA Panel on Nutrition, Novel Foods and Food Allergens (NDA)} and Turck, Dominique and Castenmiller, Jacqueline and de Henauw, Stefaan and HirschErnst, KarenIldico and Kearney, John and Knutsen, Helle Katrine and Maciuk, Alexandre and Mangelsdorf, Inge and McArdle, Harry J and Pelaez, Carmen and Pentieva, Kristina and Siani, Alfonso and Thies, Frank and Tsabouri, Sophia and Vinceti, Marco and Aggett, Peter and FairweatherTait, Susan and Martin, Ambroise and Przyrembel, Hildegard and de SesmaisonsLecarr{\' e}, Agn{\` e}s and Naska, Androniki},
+	author = {{EFSA Panel on Nutrition, Novel Foods and Food Allergens (NDA)} and Turck, Dominique and Castenmiller, Jacqueline and de Henauw, Stefaan and HirschErnst, KarenIldico and Kearney, John and Knutsen, Helle Katrine and Maciuk, Alexandre and Mangelsdorf, Inge and McArdle, Harry J and Pelaez, Carmen and Pentieva, Kristina and Siani, Alfonso and Thies, Frank and Tsabouri, Sophia and Vinceti, Marco and Aggett, Peter and FairweatherTait, Susan and Martin, Ambroise and Przyrembel, Hildegard and de SesmaisonsLecarr{\' e}, Agn{\` e}s and Naska, Androniki},
```

Comment:
Since we bumped the `citation-js` version, this must just be a small side effect; different treatment of whitespace. For what it's worth, the new space is a normal space, whereas the old "space" was a weird null character. So I think this is an improvement.